src/Imp.lidr

```diff
@@ -601,7 +601,7 @@ together with a declaration of what the notation means.
 >   E_APlus : e1 \\ n1 -> e2 \\ n2 -> n = n1 + n2 ->
 >             (APlus0 e1 e2) \\ n
 
-\todo[inline]{We don't use \idr{-} since it requires a proof of \idr{LTE n1 n2}}
+\ \todo[inline]{We don't use \idr{-} since it requires a proof of \idr{LTE n1 n2}}
```

Comment:
The added `\ ` like this after a code block and before ` \todo` brok the PDF build for me.